swift-sdk/IterableMessaging.swift

```diff
@@ -36,33 +56,35 @@ public protocol IterableInAppManagerProtocol {
 open class DefaultInAppDelegate : IterableInAppDelegate {
     public init() {}
     
+    public func onInboxReady(messages: [IterableInAppMessage]) {
+        ITBInfo()
+    }
+    
     open func onNew(message: IterableInAppMessage) -> InAppShowResponse {
         ITBInfo()
         return .show
     }
-}
 
-@objc
-public enum IterableInAppType : Int, Codable {
-    case `default`
-    case inbox
+    public func onNew(inboxMessages: [IterableInAppMessage]) {
+        ITBInfo()
+    }
 }
 
 @objc
-public enum IterableInAppContentType : Int, Codable {
+public enum IterableContentType : Int, Codable {
```

Comment:
If this can't apply to anything else, I think we can change it back to `IterableInAppContentType` to scope it to in-app.